presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -104,6 +104,20 @@ String
 
     Example type definitions: ``varchar``, ``varchar(20)``
 
+    SQL supports various types of string literals, including simple and Unicode strings:
```

Comment:
```suggestion
    SQL supports simple and Unicode string literals:
```

Suggested editing for conciseness and precision. "Various types...including" is vague and does not add information to the sentence.